src/ui/components/shared/SettingsModal/ReplayInvitations.tsx

```diff
@@ -77,12 +79,12 @@ export default function ReplayInvitations() {
         </form>
       )}
       <div className="invitations-list">
-        {invitations.map((invite, i) => (
+        {invitations!.map((invite, i) => (
           <div className="invitations" key={i}>
-            <div className={`material-icons ${invite.invited_user?.invited && "finished"}`}>
-              {invite.invited_user?.invited ? "check_circle" : "pending"}
+            <div className={`material-icons ${invite.pending || "finished"}`}>
+              {invite.pending ? "pending" : "check_circle"}
```

Comment:
Currently, `pending` is always set to `false`, but I left the code there in case we want to bring back the ability to see the pending state in the UI.